doc/langref.html.in

```diff
@@ -2497,6 +2497,42 @@ or
 
       {#header_close#}
 
+      {#header_open|Labeled switch#}
```

Comment:
Awkward wording, and doesn't mention `break`. Maybe something like this?

> When a `switch` statement is labeled, it can be referenced from a `break` or `continue` statement.
> A `break` statement will immediately jump to the end of the `switch` block, with the entire `switch` evaluating to the `break`'s operand.
> A `continue` statement targeting a switch must have an operand, and will jump to whichever prong matches its operand *without re-evaluating the original switch expression* (maybe this part is better left out).

Note that `switch` is different from loops, which can be implicitly `continue`d or `break`d without a label, and also different from blocks, in that an unlabeled break can target a labeled switch. I'm looking for clarification on whether or not this is a bug.